### PR TITLE
docs: added reference to create commit message

### PR DESCRIPTION
**Pull request by ryanadhi**
Closes https://github.com/kawalcovid19/wargabantuwarga.com/issues/644

## Description

it will help new contributor especially new developer to understand creating good (and accepted) commit message

## Current Tasks

- [x] update docs

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Draft Pull Requests is a feature provided by GitHub as a means to communicate wi
 
 Steps to creating a Draft Pull Request:
 
-1. Commit and push your new changes into the remote repository.
+1. Commit and push your new changes into the remote repository. Please refer to [@commitlint/config-conventional](https://github.com/conventional-changelog/commitlint/tree/master/@commitlint/config-conventional) for your commit message.
```
Comment (zainfathoni) [inline CONTRIBUTING.md:44]: Please let the contributors know that they can use [commitlint.io](https://commitlint.io) to assist them in composing the commit message.

**Review by zainfathoni — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (ryanadhi) [reply CONTRIBUTING.md]: Okay. I just pushed for the additional info

**Review by zainfathoni — APPROVED** [review]
LGTM 💯 

Thanks for contributing, @ryanadhi! 🙏

**Comment by zainfathoni** [comment]
@all-contributors please add @ryanadhi for doc.

**Pull request merged**